api-internal/src/routes/accesso/auth_done.rs

```diff
@@ -83,13 +89,13 @@ pub async fn route(
                     };
 
                     let viewer_get_url = {
-                        let mut uri = Url::clone(&accesso_url);
-                        uri.set_path("/api/v0/viewer");
+                        let mut uri = AccessoPublicUrl::clone(&accesso_public_url);
+                        uri.set_path("/viewer.get");
```

Comment:
But on accesso.sova.dev should be v0:
https://api.accesso.sova.dev/v0/viewer.get